config/schedule.yml

```diff
@@ -19,3 +19,9 @@ trigger_imap_email_inboxes_job:
   cron: '*/1 * * * *'
   class: 'Inboxes::FetchImapEmailInboxesJob'
   queue: scheduled_jobs
+
+# refresh ms oauth token for the channel
+trigger_ms_oauth_token_refresh:
```

Comment:
We will be doing this step every time when we fetch the mail right ? ( in 1 minute ). In case we won't be needing to run the job separately in a schedule ?